Marharyta_Miliuk/Module 3 - Java/Week 2/NewYearPresent/src/by/epam/cdp/java/nypresent/beans/Sweets.java

```diff
@@ -0,0 +1,35 @@
+package by.epam.cdp.java.nypresent.beans;
+
+public abstract class Sweets implements Comparable <Sweets> {//implementing comparable interface to make Sweets objects comparable and achieve possibility to sort them using comparator
+	
```

Comment:
Why does Sweet  implement Comparable interface?
I suppose will  be better separate comparator from bean. for example add additional WeightComparator in Present class.
the functionality should be in one plase